experiments/scripts/debug-component.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
```

Comment:
NIT / Reminder: this would be very useful standalone outside of the repo. No need for you to act on it here though.